source-hubspot-native/source_hubspot_native/resources.py

```diff
@@ -1,16 +1,19 @@
 from datetime import datetime, UTC, timedelta
-from typing import AsyncGenerator, Awaitable, Iterable
+from typing import AsyncGenerator, Awaitable, Iterable, Dict
 from logging import Logger
 import functools
 
 from estuary_cdk.flow import CaptureBinding
 from estuary_cdk.capture import Task
-from estuary_cdk.capture.common import Resource, LogCursor, PageCursor, open_binding
+#from estuary_cdk.capture.common import Resource, LogCursor, PageCursor, open_binding
```

Comment:
Why is this commented out?

Reply:
This was for code standardization, i saw that google-sheets-native was using `from estuary_cdk.capture import common` and decided that it would be best to keep everything stardard from this initial point forward, nothing important

Reply:
OK, having it removed entirely seems good, just didn't want to leave a mysterious line of commented code in. But it looks good now.